shell/opslevel/upload_prismaci.sh

```diff
@@ -46,6 +44,7 @@ while [ "$(date +%s)" -lt $ENDTIMEOUT ]; do
   if [[ ${RESULT_FILE_MESSAGE} == "Build not found" ]]; then
     continue
   fi
+  sleep 2
```

Comment:
This is the wrong place - you want to `sleep` before you re-loop, which happens at the `continue` on line 45. _That_ used to be line 47 before the sleep delete. :)

You could instead simply remove lines 43-46, since an if-continue at the end of a loop is kind of pointless ...